lib/create-organization-api.js

```diff
@@ -142,44 +137,50 @@ export default function createOrganizationApi ({ http }) {
    * .catch(console.error)
    */
   function createTeamMembership (teamId, data = {}) {
-    return http.post('teams/' + teamId + '/team_memberships', data, {headers: teamAlphaHeaders})
-      .then((response) => wrapTeamMembership(http, response.data, teamId), errorHandler)
+    return http.post('teams/' + teamId + '/team_memberships', data)
+      .then((response) => wrapTeamMembership(http, response.data), errorHandler)
   }
 
   /**
-   * Gets an Team Membership
+   * Gets an Team Membership from the team with given teamId
    * @memberof ContentfulOrganizationAPI
+   * @param {string} teamId - Team ID
    * @param {string} id - Team Membership ID
    * @return {Promise<TeamMembership>} Promise for an Team Membership
    * @example
    * const contentful = require('contentful-management')
    *
-   * client.getOrganizations()
+   * client.getOrganization('organizationId')
    * .then((organization) => organization.getTeamMembership('teamId', 'teamMembership_id'))
    * .then((teamMembership) => console.log(teamMembership))
    * .catch(console.error)
    */
   function getTeamMembership (teamId, teamMembershipId) {
-    return http.get('teams/' + teamId + '/team_memberships/' + teamMembershipId, {headers: teamAlphaHeaders})
-      .then(response => wrapTeamMembership(http, response.data, teamId), errorHandler)
+    return http.get('teams/' + teamId + '/team_memberships/' + teamMembershipId)
+      .then(response => wrapTeamMembership(http, response.data), errorHandler)
   }
 
   /**
-   * Gets an Team Membership
+   * Get all Team Memberships for the team
    * @memberof ContentfulOrganizationAPI
    * @param {string} id - Team Membership ID
```

Comment:
This is now "opts" config object, with optional `teamID` key :) (let's include both those pieces of info in the docstring)